### ISSUE TITLE
How to set different learning rate for different groups of parameters in fine-tuning?

### ISSUE THREAD

**Issue by rationalspark** [modelscope/ms-swift#1271]
Can grouped learning rates be passed to swift in CLI in fine-tuning of LLMs?
If it is not supported yet, how to modify the codes to use grouped learning rates?
Thanks for your kindness help.
Best wishes.

**Comment by rationalspark** [comment]
Can somebody help me?

**Comment by tastelikefeet** [comment]
Sorry for replying late...
https://github.com/modelscope/ms-swift/blob/main/swift/trainers/mixin.py#L660
modify the code here.
And we will do a big refactor to provide the plugin ability
